Blueprints/BlueprintDefinitions/vs2022/EmptyServerless-Image/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -84,3 +85,85 @@ Deploy application
     cd "BlueprintBaseName.1/src/BlueprintBaseName.1"
     dotnet lambda deploy-serverless
 ```
+
+## Lambda Annotations
+This template now uses [Lambda Annotations](https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md) to bridge the gap between the Lambda programming model and a more idiomatic .NET model.
+
+This automatically handles reading parameters from an APIGatewayProxyRequest and returning an APIGatewayProxyResponse. 
+
+It also generates the function resources in a JSON or YAML CloudFormation template based on your function definitions, and keeps them updated.
+
+### Using Annotations without API Gateway
+If you still wish to use Lambda Annotations for the CloudFormation template syncing without taking advantage of the integration with API Gateway, replace your function definition with the following:
+```
+    [LambdaFunction(PackageType = LambdaPackageType.Image, Policies = "AWSLambdaBasicExecutionRole", MemorySize = 256, Timeout = 30)]
+    public string Get(ILambdaContext context)
+    {
+        context.Logger.LogInformation("Handling the request");
+
+        return "Hello AWS Serverless";
+    }
+```
+
+You must also remove the "ApiURL" from the "Outputs" section in serverless.template as that is an API Gateway feature.
```

Comment:
This sentence and the next sentence aren't really anything to do with annotations. This is just what you need to do once you delete the starting content. Same thing with the original content of the empty template. I would suggest removing them.

Reply:
Removed